packages/enzyme-adapter-react-16.1/src/ReactSixteenOneAdapter.js

```diff
@@ -295,12 +295,17 @@ class ReactSixteenOneAdapter extends EnzymeAdapter {
         return instance ? toTree(instance._reactInternalFiber).rendered : null;
       },
       simulateError(nodeHierarchy, rootNode, error) {
-        const { instance: catchingInstance } = nodeHierarchy
-          .find(x => x.instance && x.instance.componentDidCatch) || {};
+        const isErrorBoundary = ({ instance: elInstance, type }) => (
```

Comment:
why is the `elInstance` rename needed?

Reply:
ESLint rule `no-shadow` wouldn't allow plain `instance`.